apt-cacher-ng.service.patch by Jan Was <janek.jan@gmail.com>

Fix paths for socket, pidfile and binaries in default systemd service file.

diff -urNad systemd/apt-cacher-ng.service systemd/apt-cacher-ng.service.arch
--- systemd/apt-cacher-ng.service	2013-08-20 00:44:35.000000000 +0200
+++ systemd/apt-cacher-ng.service.arch	2013-08-20 23:21:50.018503052 +0200
@@ -4,11 +4,11 @@
 After=network.target
 
 [Service]
-ExecStart=/usr/sbin/apt-cacher-ng SocketPath=/var/run/apt-cacher-ng/socket -c /etc/apt-cacher-ng ForeGround=1
+ExecStart=/usr/bin/apt-cacher-ng SocketPath=/run/apt-cacher-ng/socket -c /etc/apt-cacher-ng ForeGround=1
 # this Pre/Post commands can be removed together with the SocketPath option if
 # the bridge functionality is not needed
-ExecStartPre=/usr/bin/install -d --mode=0755 -o apt-cacher-ng -g apt-cacher-ng /var/run/apt-cacher-ng
-ExecStopPost=/bin/rm -f /var/run/apt-cacher-ng/*
+ExecStartPre=/usr/bin/install -d --mode=0755 -o apt-cacher-ng -g apt-cacher-ng /run/apt-cacher-ng
+ExecStopPost=/bin/rm -f /run/apt-cacher-ng/*
 User=apt-cacher-ng
 Group=apt-cacher-ng
 

